ftplugin/erlang.vim

```diff
@@ -30,6 +30,26 @@ setlocal commentstring=%%s
 
 setlocal formatoptions+=ro
 
+if get(g:, 'erlang_extend_path', 1)
+" typical erlang paths
+  let &l:path = join(['include', 'src', 'test', &g:path], ',')
+  " typical erlang.mk paths
+  let &l:path = join([
+        \ 'deps/*/include',
+        \ 'deps/*/src',
+        \ 'deps/*/test',
+        \ 'deps/*/apps/*/include',
+        \ 'deps/*/apps/*/src',
+        \ &l:path], ',')
+  " typical rebar3 paths
+  let &l:path = join([
+        \ 'apps/*/include',
+        \ 'apps/*/src',
```

Comment:
Those are also typical Erlang.mk paths, it's just RabbitMQ doing things weird. :-) See https://github.com/rabbitmq/rabbitmq-server/pull/3133

The apps/ and deps/ come from rebar2 which both Erlang.mk and rebar3 inherit.

Reply:
`deps` isn't RabbitMQ-specific with regard to `erlang.mk`, correct?

Reply:
OK I said something a bit incorrect and I will clarify.

* deps/ is something from rebar2 that Erlang.mk reuses and is not RabbitMQ specific.
* apps/ is something from rebar2 that both Erlang.mk and rebar3 support.
* The way RabbitMQ uses deps is something specific to RabbitMQ. RabbitMQ puts in deps/ what should be under apps/ (as well as the normal deps that should be under deps/).

So what I am trying to say here is that apps/ is not something specific to rebar3, it is something that rebar2/rebar3/Erlang.mk all support, and this is what RabbitMQ *should* use (but currently doesn't).

Reply:
Thanks for the clarification @lhoguin. I just wanted to ensure I wasn't proposing a RabbitMQ-specific change in this PR.